lib/audio_pool.dart

```diff
@@ -27,9 +27,7 @@ class AudioPool {
     this.maxPlayers = 1,
     this.minPlayers = 1,
     String prefix = 'assets/audio/sfx/',
-  }) {
-    cache = AudioCache(prefix: prefix);
-  }
+  }): cache = AudioCache(prefix: prefix);
```

Comment:
I suspect this file has to be formatted. 

```suggestion
  }) : cache = AudioCache(prefix: prefix);
```